regression/checkpr.sh

```diff
@@ -99,9 +99,9 @@ case $project in
     print_use; exit 1 ;;
 esac
 
-# Restrict the use of ccscs7.
+# Restrict the use of ccscs[27].
```

Comment:
1. This restriction is actually disabled (commented out).
2. I don't want devs using ccscs2 because it is already loaded with normal PR checks.
3. I don't want devs using ccscs7 because tests are failing when run from that machine.